test/test-applications/integrations/Samples.StartupHook/Samples.StartupHook.csproj

```diff
@@ -0,0 +1,6 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.0-rc.2.21480.5" />
```

Comment:
isn't it some left-over that can be removed?

Reply:
That dependency will likely also be relevant once we start to add support for OTel logging features, because the SDK relies on ILogger support to route messages through the SDK.

Reply:
I propose to do the same as for DiagnosticSource (and mention the issue workarounds in troubleshooting docs) and create an issue as you propose.